settings.gradle.kts

```diff
@@ -219,6 +220,7 @@ dependencyResolutionManagement {
             library("androidx-startup", "androidx.startup:startup-runtime:$androidxStartupVersion")
             library("androidx-viewmodel-compose", "androidx.lifecycle:lifecycle-viewmodel-compose:$androidxLifecycleVersion")
             library("androidx-workmanager", "androidx.work:work-runtime-ktx:$androidxWorkManagerVersion")
+            library("androidx-browser", "androidx.browser:browser:$androidxBrowserVersion")
```

Comment:
We should replace or update our `WebBrowserUtil` when adding this new component. Let's file a follow-up ticket for this.

Reply:
Agree. This androidx-browser is great!

Reply:
https://github.com/Electric-Coin-Company/zashi-android/issues/1581